tests/i18n/test_compilation.py

```diff
@@ -47,9 +51,11 @@ def test_no_write_access(self):
         old_mode = os.stat(mo_file_en).st_mode
         os.chmod(mo_file_en, stat.S_IREAD)
         try:
-            call_command('compilemessages', locale=['en'], stderr=err_buffer, verbosity=0)
-            err = err_buffer.getvalue()
-            self.assertIn("not writable location", err)
+            with self.assertRaisesMessage(CommandError, 'compilemessages generated one or more errors'):
+                call_command(
```

Comment:
single line looks okay here and in the next test (it's shorter than the previous line, at least)